src/pages/Profile/components/UserProfile/UserActions/UserActions.Styles.ts

```diff
@@ -1,8 +1,33 @@
 import styled from "styled-components"
 
 export const UserActionLayout = styled.section`
+  width: 100%;
+  height: 4rem;
   margin-bottom: 2rem;
   display: flex;
-  gap: 2rem;
   justify-content: center;
 `
+
+export const UserActionButton = styled.button<{ isSelected?: boolean }>`
+  position: absolute;
+  padding: 0.8rem 1.2rem;
+  margin: 0 1rem;
+  font-size: ${({ theme }) => theme.fontSizes.semiLarge};
+  background-color: ${({ theme, isSelected }) =>
+    isSelected ? theme.colors.point_alt : theme.colors.point};
+  border-radius: ${({ theme }) => theme.radius.size8};
+  user-select: none;
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.point_alt};
+  }
+`
+export const UserFolloActionwButton = styled(UserActionButton)`
+  right: 50%;
+`
+
+export const UserSendDMActionButton = styled(UserActionButton)`
+  left: 50%;
+  transform: translateX(2%);
+`
```

Comment:
[ P3 ] 스타일이 항상 어렵네요 ㅠㅠ 
저는 개인적인 취향은 현재 DM 보내기와 팔로우 버튼의 width 값이 동일하면 좋겠다는 생각이 들더라구요.
```suggestion
export const UserFolloActionwButton = styled(UserActionButton)`
  width: 9rem;
  right: 50%;
`

export const UserSendDMActionButton = styled(UserActionButton)`
  width: 9rem;
  left: 50%;
  transform: translateX(2%);
`
```
![image](https://github.com/prgrms-fe-devcourse/FEDC5_Yap_YUNHO/assets/127748428/d8dc5d10-2bc0-47cd-8f0f-071ebb4d09ea)

요렇게는 어떠실까요?

Reply:
혹시 현재 버튼들이 재사용되셔서 별도 prop으로 정하신거면 좋은것같습니다!